2015/music-and-measure-theory/german/sentence_translations.json

```diff
@@ -323,19 +323,19 @@
    339.88
   ]
  },
- {
-  "input": "By open interval, I just mean the continuous stretch of real numbers strictly greater than some number a, but strictly less than some other number b, where b is of course greater than My challenge to you involves covering all of the rational numbers between 0 and 1 with open intervals.",
-  "model": "nmt",
-  "translatedText": "Mit offenem Intervall meine ich einfach die kontinuierliche Strecke reeller Zahlen, die strikt größer als eine Zahl a, aber strikt kleiner als eine andere Zahl b sind, wobei b natürlich größer als ist. Meine Herausforderung an Sie besteht darin, alle rationalen Zahlen zwischen 0 und abzudecken 1 mit offenen Intervallen.",
-  "time_range": [
-   341.32,
-   358.18
-  ]
- },
+  {
+    "input": "By open interval, I just mean the continuous stretch of real numbers strictly greater than some number a, but strictly less than some other number b, where b is of course greater than My challenge to you involves covering all of the rational numbers between 0 and 1 with open intervals.",
```

Comment:
@3b1b The english text looks incomplete. 
where b is of course greater than ... than what? a?

Reply:
You're right, that is missing an "a"